autopush/db.py

```diff
@@ -146,20 +163,21 @@ def make_rotating_tablename(prefix, delta=0, date=None):
 
 def create_rotating_message_table(prefix="message", delta=0, date=None,
                                   read_throughput=5,
-                                  write_throughput=5):
-    # type: (str, int, Optional[datetime.date], int, int) -> Table
+                                  write_throughput=5,
+                                  session=None):
```

Comment:
Above we have 'session' which is a boto session.Session, yet here its a session.resource which is not the same. Here it's actually a DynamoDB resource created with the boto session.Session.